### PR TITLE
ENG-1544 unexpected item counts in assets + included other entities to avoid conflict of pagination data

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **jeffgo10**

If they are not already assigned, you can assign the PR to them by writing `/assign @jeffgo10` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/assets/actions.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ export const fetchAssets = (page, params) => dispatch => new Promise((resolve) =
       response.json().then((json) => {
         if (response.ok) {
           dispatch(setAssets(json.payload));
-          dispatch(setPage(json.metaData));
+          dispatch(setPage(json.metaData, 'assets'));
```
Comment (gidesan) [inline src/state/assets/actions.js:106]: Please let's use constants, for this one and for the similar ones in this PR.

File: src/ui/assets/AssetsListContainer.js [hunk 2]
```diff
@@ -23,36 +23,39 @@ import {
...
-  sort: getListFilterParams(state).sorting || {},
-  lastPage: getLastPage(state),
-  pageSize: getPageSize(state),
-  totalItems: getTotalItems(state),
-  page: getCurrentPage(state),
-  categories: getCategoryTree(state),
-  categoryTreeFetched: getCategoryTreeFetched(state),
-});
+export const mapStateToProps = (state) => {
+  const {
```
Comment (gidesan) [inline src/ui/assets/AssetsListContainer.js:39]: Please put the "conditional" logic like `getPagination(state, 'assets') || getPagination(state);` straight in the selector if really needed (do we really have scenarios where `getPagination(state, 'assets')` is null and we need to fallback to `getPagination(state);`?)

**Review by gidesan — CHANGES_REQUESTED** [review]

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/318#pullrequestreview-539945921" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**